### ISSUE TITLE
[SUGGESTION] Add scaling to PokéDollars and Dungeon Tokens quests

### ISSUE THREAD

**Issue by Kays** [pokeclicker/pokeclicker#2149]
**Current Behavior**

1. Gain PokéDollars quest does not scale.
2. Gain dungeon tokens quest does not scale.

**Improved Behavior**

1. The amount of PokéDollars to gain and/or rewards should scale with highest region unlocked. Seeing a quest asking me to gain 50k+ PokéDollars in Kanto, upon unlocking Daily Quests (after Pewter Gym) is an insane amount to get. Meanwhile, I can get it within a second in Alola. So I think the amount of PokéDollars to gain needs to be scaled by highest region with its current rewards, or both the rewards and the amount need to be scaled up if 50k+ PokéDollars is reasonable for Kanto.
2. Similar to 1. Getting 4,943 dungeon tokens is really hard when the player has just started the game and the rewards is terrible at 477 QP. Compare it to beating Viridian Forest 15 times for 3,315 QP.

**Reason for change**

They scales very well the further you go, which is fine. I don't hate that some quests feel easier to complete as you level up or unlock more regions. But it feels weird that the bar is set so high initially (Kanto) to being a freebie by mid-game. I think they should scale, at least a little bit, so it's not a "click start, wait one second, click claim".

**Detailed Description**

**Screenshots:**

Sample of quests in Kanto, upon unlocking Daily Quests
![12-051109_2vhBjchshn](https://user-images.githubusercontent.com/505500/168043777-278c0be4-a089-4bc0-b611-e971f065c976.png)

**Comment by Kays** [comment]
I don't think it is a duplicate. That suggestion was to scale quests (all quests) by quest level. I'm suggesting scale these two specific quests by highest region unlocked because money and DT have a cap based on the region, route, and achievement bonus whereas quest levels are different.

**Comment by amative1** [comment]
Rather than submitting a new suggestion for that though, you should add your idea as a comment to the existing suggestion. If a change is ultimately made to quest scaling, it's going to be based on that discussion that has already happened. Better to voice this opinion there and see if it can be worked into the overall idea.

**Comment by Kays** [comment]
Okay, no problem. I did reply to that with my perspective on that change, although slightly not the same as this suggestion, but the author of that ticket didn't like that I intruded on his/her idea with a different one so I didn't want to comment in it further.

![13-002026_oIwqE03thd](https://user-images.githubusercontent.com/505500/168210618-7bfd9c0e-51e2-4282-a713-1662da9d29e4.png)
